graphql/schemabuilder/batch.go

```diff
@@ -11,7 +11,11 @@ import (
 
 // buildBatchFunction corresponds to buildFunction for a batchFieldFunc
 func (sb *schemaBuilder) buildBatchFunctionWithFallback(typ reflect.Type, m *method) (*graphql.Field, error) {
-	fallbackField, fallbackFuncCtx, err := sb.buildFunctionAndFuncCtx(typ, &method{Fn: m.BatchArgs.FallbackFunc, MarkedNonNullable: m.MarkedNonNullable})
+	fallbackField, fallbackFuncCtx, err := sb.buildFunctionAndFuncCtx(typ, &method{
```

Comment:
is there a way to make this a blacklist instead of a whitelist for options to pass through?

Reply:
hmm, that sounds do-able.  Can look into that after getting the expensive stuff working again :)